airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -64,6 +64,8 @@ class SnowflakeHook(DbApiHook):
     :param session_parameters: You can set session-level parameters at
         the time you connect to Snowflake
     :type session_parameters: Optional[dict]
+    :param query_ids:
```

Comment:
There should be a description here. The API docs for this hook are built from the docstring and we shouldn't have a parameter without a description.

Reply:
I think this is simply wrong? The hook has an attribute named `query_ids`, but does not accept `query_ids` as an argument.

Reply:
Oh duh, you’re right, that was silly of me to even suggest.